src/main/java/com/spotinst/sdkjava/model/repo/ocean/ecs/SpotOceanEcsClusterRepo.java

```diff
@@ -0,0 +1,107 @@
+package com.spotinst.sdkjava.model.repo.ocean.ecs;
+
+import com.spotinst.sdkjava.exception.ExceptionHelper;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.bl.ocean.ecs.OceanEcsCluster;
+import com.spotinst.sdkjava.model.ISpotOceanEcsClusterRepo;
+import com.spotinst.sdkjava.model.api.ocean.ecs.*;
+import com.spotinst.sdkjava.model.RepoGenericResponse;
+import com.spotinst.sdkjava.model.service.ocean.ecs.SpotOceanEcsClusterService;
+import com.spotinst.sdkjava.model.converters.ocean.ecs.OceanEcsConverter;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SpotOceanEcsClusterRepo implements ISpotOceanEcsClusterRepo {
+
+    @Override
+    public RepoGenericResponse<OceanEcsCluster> create(OceanEcsCluster clusterToCreate, String authToken, String account) {
+
+        RepoGenericResponse<OceanEcsCluster> retVal;
+
+        try {
+            ApiOceanEcsCluster apiOceanEcsClusterToCreate = OceanEcsConverter.toDal(clusterToCreate);
+            ApiOceanEcsCluster apiCreatedCluster = SpotOceanEcsClusterService.createEcsCluster(apiOceanEcsClusterToCreate, authToken, account);
+
+            OceanEcsCluster createdOceanEcsCluster = OceanEcsConverter.toBl(apiCreatedCluster);
+            retVal = new RepoGenericResponse<>(createdOceanEcsCluster);
+        }
+        catch (SpotinstHttpException ex) {
+            retVal = ExceptionHelper.handleHttpException(ex);
+        }
+
+        return retVal;
+    }
+
+    @Override
+    public RepoGenericResponse<Boolean> update(String clusterId, OceanEcsCluster clusterUpdate, String authToken,
+                                               String account) {
+        RepoGenericResponse<Boolean> retVal;
+
+        ApiOceanEcsCluster apiOceanEcsCluster = OceanEcsConverter.toDal(clusterUpdate);
+
+        try {
+            Boolean success = SpotOceanEcsClusterService
```

Comment:
rename to isSucceeded (boolean should start with a question : is/did/etc

Reply:
Incorporated